app/templates/dayview.html

```diff
@@ -1,75 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Were these two files deleted on purpose?

Reply:
Yes, Ode moved them to "app/templates/partials/calendar/event/edit_event_details_tab.html" and to "app/templates/calendar_day_view.html" so I made the needed changes and deleted them in order to prevent confusion